web/src/PredictImpactMode.svelte

```diff
@@ -57,6 +94,10 @@
       Thicker roads have more traffic after edits, relative to the max count for
       any road: {impactGj.max_count.toLocaleString()}
     </p>
+    <label>
+      Only show roads with at least this much traffic before or after
+      <input type="number" min={0} bind:value={$minImpactCount} />
```

Comment:
What are the units here - "trips per day"? Might be worth labeling.

Reply:
Done, though I expect we'll adjust this wording more to de-emphasize exact counts. Maybe this filter in the first place is not quite right.